src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__field calendar__field--start-day_mon"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length_31 calendar__field--start-day_mon">
      <div class="calendar__field"></div>
```

it must be a calendar class 😉